examples/erc1155/tests/erc1155.rs

```diff
@@ -2173,3 +2177,303 @@ async fn support_interface(alice: Account) -> eyre::Result<()> {
 
     Ok(())
 }
+
+// ============================================================================
+// Integration Tests: ERC-1155 Pausable Extension
+// ============================================================================
+
+#[e2e::test]
+async fn pauses(alice: Account) -> eyre::Result<()> {
+    let contract_addr = alice
+        .as_deployer()
+        .with_default_constructor::<constructorCall>()
+        .deploy()
+        .await?
+        .address()?;
+
+    let contract = Erc1155::new(contract_addr, &alice.wallet);
+
+    let receipt = receipt!(contract.pause())?;
+
+    assert!(receipt.emits(Erc1155::Paused { account: alice.address() }));
+
+    let paused = contract.paused().call().await?.paused;
+
+    assert!(paused);
+
+    Ok(())
+}
+
+#[e2e::test]
+async fn unpauses(alice: Account) -> eyre::Result<()> {
+    let contract_addr = alice
+        .as_deployer()
+        .with_default_constructor::<constructorCall>()
+        .deploy()
+        .await?
+        .address()?;
+
+    let contract = Erc1155::new(contract_addr, &alice.wallet);
+
+    let _ = watch!(contract.pause())?;
+
+    let receipt = receipt!(contract.unpause())?;
+
+    assert!(receipt.emits(Erc1155::Unpaused { account: alice.address() }));
+
+    let paused = contract.paused().call().await?.paused;
+
+    assert_eq!(false, paused);
+
+    Ok(())
+}
+
+#[e2e::test]
+async fn mint_reverts_in_paused_state(alice: Account) -> eyre::Result<()> {
+    let contract_addr = alice
+        .as_deployer()
+        .with_default_constructor::<constructorCall>()
+        .deploy()
+        .await?
+        .address()?;
+    let contract = Erc1155::new(contract_addr, &alice.wallet);
+
+    let alice_addr = alice.address();
+    let token_id = random_token_ids(1)[0];
+    let value = random_values(1)[0];
+
+    let _ = watch!(contract.pause())?;
+
+    let err = send!(contract.mint(
+        alice_addr,
+        token_id,
+        value,
+        vec![0, 1, 2, 3].into()
+    ))
+    .expect_err("should return EnforcedPause");
```

Comment:
```suggestion
    .expect_err("should return `EnforcedPause`");
```
nit: there are other occurrences in other tests